cypress/integration/move.spec.js

```diff
@@ -29,148 +29,105 @@ describe("Move flow", () => {
     TEST_REPO_RESOURCE_CATEGORY_NAME_1
   )
 
+  beforeEach(() => {
+    cy.setCookie(COOKIE_NAME, COOKIE_VALUE)
+    window.localStorage.setItem("userId", "test")
+    waitForDom()
+  })
```

Comment:
I think this might be useful enough to move into utils, since ideally we should be using this for every test?